backend/models/lesson.model.ts

```diff
@@ -0,0 +1,36 @@
+import mongoose, { Schema, Document } from "mongoose";
+
+export interface Lesson extends Document {
+  id: string;
+  course: string;
+  title: string;
+  description: string;
+  image: string;
+  content: [Object];
+}
+
+const LessonSchema: Schema = new Schema({
+  course: {
+    type: String,
+    required: true,
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: false,
+  },
+  image: {
+    type: String,
+    required: false,
+  },
+  content: {
+    type: [Object],
```

Comment:
I don't think `Object` is the right way to define this in Mongoose? Mongoose has certain [SchemaTypes](https://mongoosejs.com/docs/schematypes.html#mixed) and I think the Mixed SchemaType is best for representing an object we haven't defined yet.